### DISCUSSION TITLE
run multiple browsers at the same time & keep them open

### DISCUSSION THREAD

**Discussion by singersbalm** [FlareSolverr/FlareSolverr#1249] [Ideas]
I ran it without HEADLESS and noticed that chrome starts every time on a request and closes when finished. Wouldn't it make more sense to start n browsers and never close them? If the concurrent requests exceed n start a new browser. close these browsers after a some idle time.

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr#-sessionscreate
